.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+**Thank you for contributing to vis.js!!**
+
+Please make sure to check the following requirements before creating a pull request:
+
+* [ ] All Pull-Requests must be to the [develop branch](https://github.com/almende/vis/tree/develop). Pull-Request to `master` will be closed!
+* [ ] Make sure your changes are based on the latest version of the [develop branch](https://github.com/almende/vis/tree/develop). (Use e.g. `git fetch && git rebase origin develop` to update you feature-branch.)
+* [ ] Provide an additional or updated examples to demonstrate your changes or new features.
+* [ ] Update the documentation if you introduced new behavior.
+* [ ] Reference issue numbers of issues that your pull request addresses. (If you write something like `fixes #1781` in your git commit message this issue gets closed automatically by merging your pull-request.)
+* [ ] Expect review comments and change requests by reviewer. (Sometimes we are quite picky :wink:)
+* [ ] Delete this check-liste from the actual pull request. (This is just for your information!)
```

Comment:
I suggest to remove `This is just for your information!`. It's obvious. However, what about making the sentence bold?